test/helpers/balancer.js

```diff
@@ -0,0 +1,52 @@
+const LBPArtifact = require("../../imports/LiquidityBootstrappingPool.json");
+const LBPFactoryArtifact = require('../../imports/LiquidityBootstrappingPoolFactory.json');
+const VaultArtifact = require("../../imports/Vault.json");
+const AuthorizerArtifact = require("../../imports/Authorizer.json");
+const { constants } = require('@openzeppelin/test-helpers');
+const { ethers } = require("hardhat");
+
+const LBPFactory = async (setup) => {
+    const LBPFactory_Factory = new ethers.ContractFactory(LBPFactoryArtifact.abi, LBPFactoryArtifact.bytecode, setup.roles.root);
+    const lbpFactory = await LBPFactory_Factory.deploy(setup.vault.address);
+    return lbpFactory;
+}
+
+const Vault = async (setup) => {
+
+    const pauseWindowDuration = 0;
+    const bufferPeriodDuration = 0;
+
+    const WETH_Factory = await ethers.getContractFactory(
+		"ERC20Mock",
+		setup.roles.root
```

Comment:
I feel, we should keep it, we should in test explicitly define the caller. It helps debugging. One time, I was confused because of this and took me 2 hours to realize this. ;)

Reply:
I think I am agreeing with Fabi, because it would make things more cleaner if we don't have to defines root all the time.